juriscraper/opinions/united_states/state/arizctapp_div_2.py

```diff
@@ -1,56 +1,48 @@
-# Scraper for Court of Appeals of Arizona, Division 2
-# CourtID: arizctapp_div_2
-# Court Short Name: arizctapp_div_2
-# Author: Andrei Chelaru
-# Reviewer: mlr
-# Date created: 23 July 2014
-
-
-import re
-from datetime import datetime
+"""
+Scraper for Court of Appeals of Arizona, Division 2
+CourtID: arizctapp_div_2
+Court Short Name: arizctapp_div_2
+Author: Andrei Chelaru
+Reviewer: mlr
+History:
+    2014-07-23: Created by Andrei Chelaru
+    2021-12-10: URL changed to recent opinions page
+"""
```

Comment:
Please add your username or name so we can keep track of who knows what.

Reply:
Done